build.gradle

```diff
@@ -48,6 +48,10 @@ jar {
         attributes ("Main-Class" : "io.logz.jmx2graphite.Jmx2GraphiteJolokia",
                     "Premain-Class": "io.logz.jmx2graphite.Jmx2GraphiteJavaAgent")
     }
+
+    from {
+        configurations.compile.collect { it.isDirectory() ? it : zipTree(it) }
```

Comment:
What's the zipTree for?
Any chance you can convert this project to Maven?

Reply:
It's for making a fat-jar when building to encapsulate the required dependencies in the created jar.